backend/app/alembic/versions/add_prompt_management_tables.py

```
@@ -0,0 +1,98 @@
+        sa.Column('id', postgresql.UUID(), nullable=False),
+        sa.Column('prompt_id', postgresql.UUID(), nullable=False),
+        sa.Column('version', sa.Integer(), nullable=False),
+        sa.Column('content', sa.Text(), nullable=False),
+        sa.Column('input_vars', sa.JSON(), nullable=True),
+        sa.Column('created_by', postgresql.UUID(), nullable=False),
+        sa.Column('created_at', sa.DateTime(), nullable=False, server_default=sa.text('CURRENT_TIMESTAMP')),
+        sa.Column('change_notes', sa.String(), nullable=True),
+        sa.ForeignKeyConstraint(['prompt_id'], ['prompts.id'], ondelete='CASCADE'),
+        sa.PrimaryKeyConstraint('id')
+    )
+    
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)